tests/ImageSharp.Tests/Formats/GeneralFormatTests.cs

```diff
@@ -111,26 +111,27 @@ public void ImageCanConvertFormat()
 
             foreach (TestFile file in Files)
             {
-                Image image = file.CreateImage();
-
-                using (FileStream output = File.OpenWrite($"{path}/{file.FileNameWithoutExtension}.gif"))
+                using (Image image = file.CreateImage())
                 {
-                    image.SaveAsGif(output);
-                }
+                    using (FileStream output = File.OpenWrite($"{path}/{file.FileNameWithoutExtension}.bmp"))
```

Comment:
Tests target netstandard1.1 and net4.5.1 just now. Won't switch till we move to VS2017